### PR TITLE
Revert "PLANET-5142 Add Cloudflare's Image Optimization url twig filter

### PR DESCRIPTION
**Pull request by comzeradd**
This reverts commit 5907e6458afac97547ae4724b147c2d34c6f6093
This reverts commit 8b80668136c75bebf3f168647434dff20115cbe8

The counterpart of greenpeace/planet4-plugin-gutenberg-blocks#448

### CODE REVIEW

File: tests/test-p4-master-site.php [hunk 1]
```diff
@@ -16,42 +16,45 @@
...
 	 */
-	public function testImgToCloudflare( array $params, string $expected ): void {
+	public function testMakeContentImageLazyLoad( string $content, ?string $expected ): void {
 		/** @var MasterSite $p4 */
 		$p4 = $this->getMockBuilder( MasterSite::class )
-		->disableOriginalConstructor()
-		->setMethodsExcept( [ 'img_to_cloudflare' ] )
-		->getMock();
+			->disableOriginalConstructor()
+			->setMethodsExcept( [ 'make_content_images_lazyload' ] )
```
Comment (lithrel) [inline tests/test-p4-master-site.php:51]: Not sure the revert worked properly here, you should drop all tests, this function was removed in https://github.com/greenpeace/planet4-master-theme/pull/1176 (by you :grin:  https://github.com/greenpeace/planet4-master-theme/commit/1959090029992a5b98d47e001899ab9234dc731a#diff-d5a71432d6e94abc1260a448df306d8dfc76d15fa4047646a4d133c38cc131d7L1127 )

**Review by lithrel — CHANGES_REQUESTED** [review]

File: tests/test-p4-master-site.php (hunk 1 above)
Comment (comzeradd) [reply tests/test-p4-master-site.php:51]: Oh yes. I reverted the two commits, then squashed but something went wrong when resolving the conflicts.

Comment (comzeradd) [reply tests/test-p4-master-site.php]: Commit amended

**Review by lithrel — APPROVED** [review]
:+1: 
Should we create a ticket for a patch to remove the feature key off the db ?

**Comment by comzeradd** [comment]
> Should we create a ticket for a patch to remove the feature key off the db ?

Indeed. I’ll create a post-deploy script for that 👍

**Pull request merged**